src/DataLayer/converters/tests/fixtures/Afm.fixtures.ts

```diff
@@ -901,3 +901,48 @@ export const reducedMultipleSorts: IFixture = {
         ]
     }
 };
+
+export const nativeSubTotals = {
```

Comment:
would be better to use `IFixture` interface as other fixtures in this file :)

Reply:
done